.eslintrc

```diff
@@ -4,7 +4,7 @@
     "indent": [ "error", 2 ],
     "quotes": [ "error", "single" ],
     "semi": ["error", "always"],
-    "linebreak-style": [ "error", "unix" ]
+    # "linebreak-style": [ "error", "unix" ]
```

Comment:
You should not have added this to git. It looks like a windows specific change that does not belong in the repo.